CHANGELOG.md

```diff
@@ -2,9 +2,108 @@
 
 ## [Unreleased]
 
-### Fixed
+### Changed
 
+- Ensure `cargo-fmt` tests are excluded from the root workspace [PR #5043](https://github.com/rust-lang/rustfmt/pull/5043)
+- Address various clippy and rustc warnings
+  - [PR #5040](https://github.com/rust-lang/rustfmt/pull/5040)
+  - [PR #5135](https://github.com/rust-lang/rustfmt/pull/5135)
+  - [PR #5164](https://github.com/rust-lang/rustfmt/pull/5164)
+- Minor parser cleanup [PR #5056](https://github.com/rust-lang/rustfmt/pull/5056)
+- Various README changes
+  - [PR #5048](https://github.com/rust-lang/rustfmt/pull/5048)
+  - [PR #5110](https://github.com/rust-lang/rustfmt/pull/5110)
+  - [PR #5221](https://github.com/rust-lang/rustfmt/pull/5221)
+- Use `RUSTFMT_LOG` env var instead of `RUST_LOG` to configure rustfmt log output [PR #5051](https://github.com/rust-lang/rustfmt/pull/5051)
+- Update IntelliJ Integration documentation [PR #5049](https://github.com/rust-lang/rustfmt/pull/5049)
+- Also apply `empty_item_single_line=true` to trait definitions [#5047](https://github.com/rust-lang/rustfmt/issues/5047)
+- Dedupe and simplify type alias formatting [PR #5068](https://github.com/rust-lang/rustfmt/pull/5068)
+- Dedupe associated item visitation [PR #5069](https://github.com/rust-lang/rustfmt/pull/5069)
+- Dedupe and clean up Impl handling [PR #5087](https://github.com/rust-lang/rustfmt/pull/5087)
+- Update naming conventions for AST structures [PR #5070](https://github.com/rust-lang/rustfmt/pull/5070)
+- Update how the fn params span is calculated to avoid issues with `T: Fn()` bounds [PR #5121](https://github.com/rust-lang/rustfmt/pull/5121)
```

Comment:
Let's remove this too, there's no need to incorporate subtree syncs nor callout changes made to rustc libs that accordingly required minor changes to rustfmt source